util/perl/OpenSSL/paramnames.pm

```diff
@@ -104,6 +104,8 @@ my %params = (
 # For passing the AlgorithmIdentifier parameter in DER form
     'CIPHER_PARAM_ALGORITHM_ID_PARAMS' =>  "alg_id_param",# octet_string
     'CIPHER_PARAM_XTS_STANDARD' =>         "xts_standard",# utf8_string
+    'CIPHER_PARAM_CIPHER_THEN_MAC' =>      "cipher-then-mac",# int, 0 or 1
```

Comment:
This should be named encrypt-then-mac as that is the more commonly used term being used also in the algorithm name, etc.

Also cipher-then-mac is true only for the encryption side. When decrypting the order is inverted.

Reply:
Ack,  will change the name